crates/gpui/src/platform/blade/blade_renderer.rs

```diff
@@ -338,17 +343,32 @@ impl BladeRenderer {
             .create_surface_configured(window, surface_config)
             .unwrap();
 
+        // Determine the sample count based on the device's capabilities.
+        #[cfg(target_os = "macos")]
+        let mut sample_count = 1;
+        #[cfg(target_os = "macos")]
+        for &n in &[4, 2] {
+            if context.gpu.metal_device().supportsTextureSampleCount(n) {
+                sample_count = n as _;
+                break;
+            }
+        }
+
+        // TODO: Determine on non-macOS platforms, until Blade supports querying sample counts.
+        #[cfg(not(target_os = "macos"))]
+        let sample_count = 4;
```

Comment:
do you really need the query? pretty sure the sample count of `4` is universally supported on all platforms

Reply:
Oh, here is a reference:

https://developer.apple.com/documentation/metal/mtldevice/1433355-supportstexturesamplecount

| Sample Count | Devices |
| -- | -- |
| 1 | All devices |
| 2 | All iOS devices<br>All tvOS devices<br>Some macOS devices|
| 4 | All devices |
| 8 | Some macOS devices |